cadquery/occ_impl/shapes.py

```diff
@@ -1923,14 +1935,44 @@ def positions(
 
         return [self.positionAt(d, mode) for d in ds]
 
+    def sample(
+        self: Mixin1DProtocol, n: Union[int, float]
+    ) -> Tuple[List[Vector], List[float]]:
+        """
+        Sample a curve based on a number of points or deflection.
+
+        :param n: number of positions or deflection
+        :return: A list of Vectors and a list of parameters.
```

Comment:
Isn't this supposed to be "returns" in this context?

Reply:
According to the docs `:return:` https://sphinx-rtd-tutorial.readthedocs.io/en/latest/docstrings.html